app/views/services/index.html.erb

```diff
@@ -2,45 +2,35 @@
 
 <% breadcrumb :services %>
 
-<div class="page-title">
-  <h1>Local Links Manager</h1>
-</div>
+<%= render "govuk_publishing_components/components/heading", {
+  text: "Services (#{@services.count})",
+  margin_bottom: 5,
+} %>
 
-<%= render partial: 'shared/home_nav' %>
+<%
+  @table_rows = @services.map do |service|
+    [
+      { text: link_to(service.label, service_path(service, filter: 'broken_links')) },
+      { text: service.lgsl_code },
+      { text: service.broken_link_count, format: "numeric" },
+    ]
+  end
+%>
 
-<table class="table table-bordered table-with-counts table-with-controls" data-module="filterable-table">
-  <thead>
-    <tr class="table-header">
-      <th colspan='2'>
-        <div class='filter-control-full-width'>
-          <%= render partial: 'govuk_admin_template/table_filter_form', locals: { placeholder: 'Search for a service here' } %>
-        </div>
-        <div class='count'>
-          <span><%= @services.count %></span>
-          <%= 'services'.pluralize(@services.count) %>
-        </div>
-      </th>
-    </tr>
-    <tr class="table-header">
-      <th class="center">Code</th>
-      <th>Service</th>
-    </tr>
-  </thead>
-  <tbody>
-    <% @services.each do |service| %>
-      <tr>
-        <td>
-          <div class="text-center">
-            <%= service.lgsl_code %>
-          </div>
-        </td>
-        <td>
-          <%= link_to service.label, service_path(service, filter: 'broken_links'), class: 'name' %>
-          <div class="text-center count <%= singular_or_plural(service.broken_link_count) %>">
-            <span><%= service.broken_link_count %></span>
-          </div>
-        </td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
+<%= render "govuk_publishing_components/components/table", {
+  filterable: true,
+  label: "Filter Services",
+  head: [
+    {
+      text: "Service Name",
+    },
+    {
+      text: "LGSL Code",
+    },
```

Comment:
Is "LGSL" widely understood? Any need to write it out in full with the abbreviation in brackets?

Reply:
Hmm, probably not. I'll write it out.

Reply:
Oh, I think maybe we shouldn't write it out for this table, it'll make the column enormous. But I'll expand on it in the show template.